src/redisenterprise/azext_redisenterprise/tests/latest/test_redisenterprise_scenario.py

```diff
@@ -0,0 +1,118 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+import os
+from azure.cli.testsdk import ScenarioTest
+from azure.cli.testsdk import ResourceGroupPreparer
+from .example_steps import step_create
+from .example_steps import step_show
+from .example_steps import step_list
+from .example_steps import step_list2
+from .example_steps import step_database_show
+from .example_steps import step_database_list
+from .example_steps import step_database_list_keys
+from .example_steps import step_database_regenerate_key
+from .example_steps import step_delete
+from .example_steps import step_database_create
+from .example_steps import step_database_delete
+from .. import (
+    try_manual,
+    raise_if,
+    calc_coverage
+)
+
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+# Env setup_scenario1
+@try_manual
+def setup_scenario1(test, rg):
+    pass
+
+
+# Env cleanup_scenario1
+@try_manual
+def cleanup_scenario1(test, rg):
+    pass
+
+
+# Testcase: scenario1
+@try_manual
+def call_scenario1(test, rg):
+    setup_scenario1(test, rg)
+    step_create(test, rg, checks=[])
+    step_show(test, rg, checks=[])
+    step_list(test, rg, checks=[])
+    step_list2(test, rg, checks=[])
+    step_database_show(test, rg, checks=[])
+    step_database_list(test, rg, checks=[])
+    step_database_list_keys(test, rg, checks=[])
+    step_database_regenerate_key(test, rg, checks=[])
+    step_delete(test, rg, checks=[])
```

Comment:
Please add check values for each command execution result.

Reply:
Fixed